api/models/Community.js

```diff
@@ -141,6 +141,11 @@ module.exports = bookshelf.Model.extend(merge({
   },
 
   memberCount: function () {
+    // return this.get('num_members')
+    return this.users().fetch().then(x => x.length)
+  },
+
+  numMembers: function () {
```

Comment:
seems like we could just use this cached value that already exists instead of calculating it every time

Reply:
please see the function on line 203-208, `reconcileNumMembers`. After noticing the member counts from `num_members` were off by 1 for me locally, I noticed this function which tipped me off that num_members isn't necessarily reliable. We agreed to investigate this later, not resolve it now.